### DISCUSSION TITLE
how to fetch funding rate faster

### DISCUSSION THREAD

**Discussion by tiulinh** [aoki-h-jp/funding-rate-arbitrage#12] [General]
how to fetch funding rate faster. Now it is very slowly!

**Comment by aoki-h-jp** [comment]
Thank you for opening discussion and sorry for the late reply.
I'm currently using ccxt to fetch Funding Rate, but I think we can fetch it faster by using websocket of each exchange API.
These will be implemented in the future, but will be done on a best-effort basis as we need to devote effort to other projects.
